Source/SwiftyDropbox/Shared/Generated/Files.swift

```diff
@@ -409,7 +410,7 @@ open class Files {
         /// Whether to force the create to happen asynchronously.
         public let forceAsync: Bool
         public init(paths: Array<String>, autorename: Bool = false, forceAsync: Bool = false) {
-            arrayValidator(itemValidator: stringValidator(pattern: "(/(.|[\\r\\n])*)|(ns:[0-9]+(/.*)?)"))(paths)
+            arrayValidator(maxItems: 10000, itemValidator: stringValidator(pattern: "(/(.|[\\r\\n])*)|(ns:[0-9]+(/.*)?)"))(paths)
```

Comment:
do we hit this code? not sure if we want a precondition on 10k maxItems

Reply:
Based on my tracing the code, I think this is only used by `open func createFolderBatch(paths: Array<String>, autorename: Bool = false, forceAsync: Bool = false) -> RpcRequest<Files.CreateFolderBatchLaunchSerializer, VoidSerializer>` which we do not use